### DISCUSSION TITLE
Exact logger format

### DISCUSSION THREAD

**Discussion by Aijuh** [snu-quiqcl/qiwis#191] [Q&A] [answered]
Hello. I am currently assigned issue #181, implement logger app to show message from logger in qiwis.py.
This can be done by using root logger (using just logging.getLogger(), not logging.getLogger(__name__)). If we set logger in qiwis.py and logger.py both as logging.getLogger(), they have same root logger and log from qiwis.py is propagate to logger.py. In other examples' modules like poller.py or numgen.py, if we set logger in this code as child of root logger using getChild, logger from this module will propagate to root logger in qiwis.py. 
I also implement button and frame to select logger level (ex: DEBUG, INFO). In my local repository, it works well. But if it is not necessary, I will delete it. 
What I don't know is exact logger format. I just set it as fs ="%(name)s %(message)s" and I want to know exact format. Also I want to know what I missed and should be added in my code to finish issue #181. I add commit for what I done. Can I got checked by sending this Q&A?

**Comment by kangz12345** [comment, marked as answer]
What happens if:
- Log with `logging.getLogger(__name__)` in qiwis.py
- Handle `logging.getLogger()` in logger.py

The format is not important in #181. You can decide as you want. If there is some problem, the reviewers will point it out in the PR (pull request).

Please make a PR when you think you are done.
Of course you can ask more questions if you need to.

> Can I got checked by sending this Q&A?

If you mean that you wonder whether we receive notifications of your new discussion, yes we receive an email since we are _watching_ this repository.

**Comment by Aijuh** [comment]
Thank you.

If we use this code:

- Log with logging.getLogger(__name__) in qiwis.py

- Handle logging.getLogger() in logger.py

Since logger in logger.py is root logger, log in qiwis.py propagate to logger.py and log message print on the screen.
